clippy_utils/src/lib.rs

```diff
@@ -8,10 +8,12 @@
 #![allow(clippy::missing_errors_doc, clippy::missing_panics_doc, clippy::must_use_candidate)]
 // warn on the same lints as `clippy_lints`
 #![warn(trivial_casts, trivial_numeric_casts)]
+#![warn(rust_2021_compatibility)]
 // warn on lints, that are included in `rust-lang/rust`s bootstrap
 #![warn(rust_2018_idioms, unused_lifetimes)]
 // warn on rustc internal lints
 #![warn(rustc::internal)]
+#![warn(rust_2021_compatibility)]
```

Comment:
Added twice. Also I'd prefer if this would just be added to the same `warn` attribute as `rust_2018_idioms`. But I'd understand if you don't want to go over every file again, so if you don't want, just ignore this :+1: